snuba/cli/optimize.py

```diff
@@ -17,14 +15,11 @@
               help='The dataset to target')
 @click.option('--timeout', default=10000, type=int,
               help='Clickhouse connection send/receive timeout, must be long enough for OPTIMIZE to complete.')
-@click.option('--log-level', default=settings.LOG_LEVEL, help='Logging level to use.')
-def optimize(clickhouse_host, clickhouse_port, database, dataset, timeout, log_level):
+def optimize(clickhouse_host, clickhouse_port, database, dataset, timeout):
     from datetime import datetime
     from snuba.clickhouse.native import ClickhousePool
     from snuba.optimize import run_optimize, logger
 
-    logging.basicConfig(level=getattr(logging, log_level.upper()), format='%(asctime)s %(message)s')
```

Comment:
This changes the log message format. Given that the new one (with the message level) seems more correct. Do you know of any reason we did not have it. Also I suspect this will change the prod log format since this was hardcoded. Is that correct and is there any issue with that ?

Reply:
> Do you know of any reason we did not have it.

No idea.

> Also I suspect this will change the prod log format since this was hardcoded. Is that correct and is there any issue with that ?

Would need to ask @JTCunning. Ultimately he should probably be aware of this change regardless, I don't know why I didn't just add him in the first place.